CMakeLists.txt

```diff
@@ -512,13 +512,17 @@ endif()
 option(WITH_SYSTEM_BOOST "require and build with system Boost" OFF)
 
 set(BOOST_COMPONENTS
-	thread system regex random program_options date_time iostreams coroutine context)
+	thread system regex random program_options date_time iostreams coroutine)
 set(BOOST_HEADER_COMPONENTS container)
 
 if(WITH_MGR)
 	list(APPEND BOOST_COMPONENTS python)
 endif()
 
+if(WITH_RADOSGW)
```

Comment:
`if (WITH_RADOSGW_BEAST_FRONTEND)` should be sufficient here. then you don't need to change the packaging for radosgw